docs/api/en/lights/SpotLight.html

```diff
@@ -147,7 +147,7 @@ <h3>[property:Object3D target]</h3>
 			*Note*: For the target's position to be changed to anything other than the
 			default, it must be added to the [page:Scene scene] using
 			<code> scene.add( light.target ); </code>
-			This is so that the target's [page:Object3D.matrixWorld matrixWorld] gets
+			This is so that the target's [page:Object3D.matrixWorld matrixWorld] needs to be
```

Comment:
At least in English the suggestion feels not like an improvement.

You want to add `light.target` to the scene so the target's world matrix gets updated. The reasoning for adding target to the scene is stated in:

> For the target's position to be changed to anything other than the default, it must be added to the [page:Scene scene] using

Reply:
I know you already understand what I'm saying.
It's up to you whether modifications are needed and how they can be made more clearly.